README.md

```diff
@@ -2,26 +2,27 @@
 
 GOG Galaxy 2.0 Community integration for Steam.
 
-### Alpha Testers: Please see Installation
+This is a fork of the repository from FriendsOfGalaxy, intended to continue development until they resume their work.
+### Alpha Testers: Please see Installation Steps below Setup For Developers
 
 ## Setup (For Developers)
-* Download Python 3.7.9 32-bit
+* Download [Python 3.7.9 32-bit][Python379]
 * Install it with the defaults
 * Create a new virtual env:
-    - If you only have python 3.7.9<br/>
+    - If you only have Python 3.7.9<br/>
     `python -m venv .venv`
-    - IF you have multiple python versions installed (assumes you have `py` as well)<br/>
-    `py -3.7 -m venv .venv`
+    - If you have multiple Python versions installed (assumes you have `py` as well)<br/>
+    `py -m venv .venv`
 * Activate the virtual env 
   - Windows, Powershell:<br/>
   `.\.venv\Scripts\activate.ps1`
   - MacOS, terminal:<br/>
-  `.venv/Scripts/activate
+  `.venv/Scripts/activate`
 * Downgrade/Upgrade `pip`<br/>
     - If you only have python 3.7.9<br/>
-    `python -m pip install --upgrade pip==22.0.4`
```

Comment:
The old code specified that version. I believe we can put that in requirements\app.txt as a `pip==22.0.4` in the virtual environment? i'm not sure. which would remove that instruction from the list

Reply:
I think you're right, anything we can do to make the installation for devs/testers as easy as possible is worth looking into.

Reply:
done in latest commit :-)